### PR TITLE
[QRF-69] Fix: pasting slate content keeps empty line

### PR DESCRIPTION
**Pull request by aspirisen**
Before:

After:

### CODE REVIEW

File: packages/slate-editor/src/extensions/paste-slate-content/lib/withSlatePasting.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ export function withSlatePasting<T extends Editor>(editor: T) {
             const fragment = decodeSlateFragment(slateFragment);
 
             if (isValidFragment(fragment)) {
-                Transforms.insertNodes(editor, fragment);
+                Transforms.insertFragment(editor, fragment);
```
Comment (aspirisen) [inline packages/slate-editor/src/extensions/paste-slate-content/lib/withSlatePasting.ts:18]: Originally was changed here https://github.com/prezly/slate/pull/210/files#diff-7dbb892cca06713d9888d229abe8a4ca0dcff541fe479aed3efc1cdbc117d847L26

was it intended?

Comment (e1himself) [reply packages/slate-editor/src/extensions/paste-slate-content/lib/withSlatePasting.ts:18]: I believe I've changed it to fix the failing tests. Now they fail again :)

Comment (aspirisen) [reply packages/slate-editor/src/extensions/paste-slate-content/lib/withSlatePasting.ts:18]: yes, but maybe tests are wrong, `insertFragment` has been there for a long time i suppose

Comment (e1himself) [reply packages/slate-editor/src/extensions/paste-slate-content/lib/withSlatePasting.ts:18]: Maybe the tests are wrong :man_shrugging:

Comment (aspirisen) [reply packages/slate-editor/src/extensions/paste-slate-content/lib/withSlatePasting.ts]: @e1himself fixed tests

**Review by e1himself — APPROVED** [review]

**Pull request merged**